src/pipeline/activator/activator/planet/main.py

```diff
@@ -0,0 +1,275 @@
+                'sourceItemId': item_id
+            },
+            outfile
+        )
+
+
+def planet_cog_collection_id(num_bands: int = 4):
+    return f"planet-{num_bands}band-cogs"
+
+
+def get_planet_cog_collection(num_bands: int = 4):
+    collection = pystac.Collection(
+        planet_cog_collection_id(num_bands),
+        f'Planet Imagery: {source_collection_id(num_bands)}',
+        pystac.Extent(
+            pystac.SpatialExtent([[-180, -90, 180, 90]]),
+            pystac.TemporalExtent([[
+                datetime(1307, 10, 13, tzinfo=timezone.utc),
+                datetime(2063, 4, 5, tzinfo=timezone.utc),
+            ]]),
+        ),
+        stac_extensions=COG_COLLECTION_EXTENSIONS)
+    collection.links = []
+    collection.properties = {}
+    if num_bands == 4:
+        collection.properties['eo:bands'] = PLANET_BANDS[:3] + PLANET_BANDS[4:5]
+    elif num_bands == 5:
+        collection.properties['eo:bands'] = PLANET_BANDS
+    collection.properties['hsi:wavelength_min'] = 440.0
+    collection.properties['hsi:wavelength_max'] = 950.0
+
+    return collection
+
+
+def cli_parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--force", action="store_true", help="If provided, force reingest StacItem even though this it is already present in the catalog.")
+    parser.add_argument("--keep-temp-dir", action="store_true", help="If provided, script does not delete temporary directory before script exits. Useful for debugging.")
+    parser.add_argument("--num-bands", type=int, default=4, choices=[4, 5], help="The desired number of bands (PSScene4Band or REOrthoTile)")
+    parser.add_argument("--output-format", type=str, default=os.environ.get("GDAL_OUTPUT_FORMAT", "COG"))
+    parser.add_argument("--pipeline", type=str, help="JSON with instructions")
+    parser.add_argument("--pipeline-uri", type=str, help="A URI to JSON with instructions")
+    parser.add_argument("--planet-api-key", type=str)
+    parser.add_argument("--planet-api-uri", type=str, default="https://api.planet.com/data/v1/item-types/{}/items/{}/assets")
+    parser.add_argument("--planet-id", type=str, help="Planet Image ID")
+    parser.add_argument("--s3-bucket", type=str, default=os.environ.get("S3_BUCKET", "sentinel-s2-data"))  # sic
+    parser.add_argument("--s3-prefix", type=str, default=os.environ.get("S3_PREFIX", "planet-scene-cogs"))
+    parser.add_argument("--stac-api-uri", type=str, default=os.environ.get("STAC_API_URI", "http://franklin:9090"))
+    parser.add_argument("--temp-dir", type=str, default='/tmp')
+
+    parser.add_argument('--no-download', required=False, dest='download', action='store_false')
+    parser.set_defaults(download=True)
+
+    parser.add_argument('--no-upload', required=False, dest='upload', action='store_false')
+    parser.set_defaults(upload=True)
+
+    parser.add_argument('--no-update', required=False, dest='update', action='store_false')
+    parser.set_defaults(update=True)
```

Comment:
On other _activators_ configuration is handled through a separate config parser https://github.com/azavea/nasa-hyperspectral/blob/master/src/pipeline/activator/activator/aviris/config.py

It allows to parse params as arguments as well as as a json pipeline. I think the right move here or 1. to add the config.py parser or 2. reduce amount of params there